src/formats/MOL2.cpp

```diff
@@ -291,10 +314,18 @@ void MOL2Format::write(const Frame& frame) {
             resid = std::to_string(++max_resid);
         }
 
+        std::string sybyl;
+        if (frame[i].get("sybyl") && frame[i].get("sybyl")->kind() == Property::STRING) {
+            sybyl = frame[i].get("sybyl")->as_string();
+        } else {
+            warning("Sybyl type not set. Using element type instead");
```

Comment:
Should this use a warn_once instead, not to flood the output?

Reply:
Sure, it's far more likely that someone will miss all sybyl types instead of just one.